dev/TreeView/TreeView.idl

```diff
@@ -147,6 +147,13 @@ unsealed runtimeclass TreeView : Windows.UI.Xaml.Controls.Control
         static Windows.UI.Xaml.DependencyProperty ItemContainerTransitionsProperty{ get; };
         static Windows.UI.Xaml.DependencyProperty ItemsSourceProperty{ get; };
     }
+
+    [WUXC_VERSION_MUXONLY]
+    {
+        TreeViewNode SelectedNode{ get; set; };
+        Object SelectedItem{ get; set; };
+        Windows.Foundation.Collections.IVector<Object> SelectedItems{ get; };
```

Comment:
should this be IVectorView. I assume you do not want this collection to be read only.

Reply:
I think IVectorView is read only? We want the vector property to be read only (they can't swap the vector), but the content of that vector is writable and we are listening on vector changes.

Reply:
Is there a reason we don't want to enable setting the vector? Is it harder for us to support?

Reply:
I was just following the patterns from TreeView.SelectedNodes and ListView.SelectedItems. 

I don't think providing a setter will make things easier? They would have to create a collection themselves, add items into it, and then set it to TreeView, whereas they can just do TreeView.SelectedItems.Add() currently. Also, I think we've made assumptions that SelectedItems/Nodes vector will never be null. We'll have to add a bunch of null checks if opening the setter.

Reply:
Ok, makes sense. Could you follow up with Premalini about adding this new API? We need to probably have a separate GitHub issue to bring this API through API review.

Reply:
Actually we'll use #197. You can check this in with the API under "preview" and then we have to wait until it goes through API review and doc process to flip to public.

Reply:
Yeah sounds good. @predavid for context.